crates/rattler_solve/src/libsolv/util.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(Copy, Clone, Ord, PartialEq, PartialOrd, Eq)]
+pub enum PackageExtension {
+    TarBz2,
+    Conda,
+}
+
+/// Given a package filename, extracts the filename and the extension if the extension is a known
+/// package extension.
+pub fn extract_known_filename_extension(filename: &str) -> Option<(&str, PackageExtension)> {

```

Comment:
We have the `ArchiveType` thingy in `rattler_conda_types` - could we use it here?

Reply:
Either `ArchiveIdentifier`: https://github.com/mamba-org/rattler/blob/main/crates/rattler_conda_types/src/package/archive_identifier.rs or https://github.com/mamba-org/rattler/blob/main/crates/rattler_conda_types/src/package/archive_type.rs

Reply:
Good catch! `ArchiveType` is perfect here